portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/servicebuilder/ServiceBuilder.testcase

```diff
@@ -18,6 +18,41 @@ definition {
 		}
 	}
 
+	@priority = "4"
+	test ExternalDataSourceSmoke {
+		property test.assert.warning.exceptions = "true";
+		property test.name.skip.portal.instance = "ServiceBuilder#ExternalDataSourceSmoke";
+
+		var liferayHome = PropsUtil.get("liferay.home.dir.name");
+		var appServerDir = PropsUtil.get("app.server.dir");
+
+		AntCommand(
+			locator1 = "build-test.xml",
+			value1 = "replace-file -Dfile=${appServerDir}/webapps/ROOT/WEB-INF/classes/portal-ext.properties -Dtoken=jdbc.default -Dvalue=jdbc.test");
+
+		Portlet.shutdownServer();
```

Comment:
You can use property skip.start.app.server = "true"; to skip starting the portal first instead of having to restart portal